exec/cnex/global.c

```diff
@@ -186,7 +187,7 @@ void file_copy(TExecutor *exec)
 
     //BOOL r = CopyFile(filename, destination, TRUE);
     //if (!r) {
-    //    handle_error(exec, GetLastError(), destination);
+        handle_error(exec, GetLastError(), destination);
```

Comment:
What is this doing now? Does this function now always fail? Is that by design? Why is the code that does the work commented out?